### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by farishkash** [AtrumGeost/teacozy#7]
Overall met expectations.

Structurally the code is great.

Your div usage is proper, your css is efficient.
I mention comments and alt text.  Comments is to help with organization and faster debugging.

Alt Text should be used to help the visually impaired.  People tend not to realize how many blind people use the web.

Visually there is a couple issues, this is correctable with margin and padding adjustments.

The upcoming projects will be using media queries.

https://www.w3schools.com/css/css_rwd_mediaqueries.asp

See if you can add media query to this site to make it look like an app on a mobile device. It will help prepare you for the final stretch of projects.

**Comment by AtrumGeost** [comment]
Hi @farishkash ,

Thanks for your review and comments. I have done some corrections. I will take in count all your recommendations for future projects.
